deltachat-rpc-client/src/deltachat_rpc_client/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .account import Account
+from .contact import Contact
+from .deltachat import Deltachat
+from .message import Message
+from .rpc import Rpc, new_online_account, start_rpc_server
```

Comment:
`new_online_account()` doesn't seems like something that should be exposed to the user, it is only used internally for CI/tests right? I would even prefix it with `_` to avoid bothering the user with internal functions while exploring the API in ipython etc